### PR TITLE
Homework Backend and Frontend Communication

### PR DESCRIPTION
**Pull request by heofthetea**
Make sure backend connects to the frontend.
Yes, adding the frontend in the first place counts as 'connecting' in this case.

"database"? Thanks github for deleting half the comment

### CODE REVIEW

**Review by ScheerleJo — DISMISSED** [review]
Looks good to me! Nice Documentation :D

**Comment by heofthetea** [comment]
figured if we have Chris and Timo work with it they better have good info on how to use it

**Comment by heofthetea** [comment]
@ScheerleJo I'll remove the name from the Homework Entity, as that is unecessary and creates useless overhead and confusion when dealing with Homework as a child of Note

**Review by heofthetea — COMMENTED** [review]
Basst scho

File: app/src/main/java/stark/prm/project/HomeworkActivity.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        Lecture lecture = null;
+        HashMap<UUID, Lecture> lectureMap = db.getLectures();
+        for (Lecture item : lectureMap.values()) {
+            if(item.getTopic().equals(topic)) lecture = item;
+        }
+        if(lecture == null) {
+            lecture = new Lecture(module, topic);
+        }
+        
+        stark.prm.project.data.Homework homework = new stark.prm.project.data.Homework(
```
[the remark this answers is not in the document]
Comment (ScheerleJo) [reply app/src/main/java/stark/prm/project/HomeworkActivity.java]: Is done

**Pull request merged**